app/src/androidTest/java/org/mozilla/tv/firefox/integration/SearchEngineManagerFactoryTest.kt

```diff
@@ -35,4 +36,12 @@ class SearchEngineManagerFactoryTest {
         // We don't test for AMAZON_SEARCH_CODE_US_ONLY so that this test won't
         // fail when run outside of the US
     }
+
+    @Test
```

Comment:
I think this is the most important test we have: maybe we should highlight that with a comment?
```kotlin
@Test // This is a very important end-to-end test
```

? Might be overboard though: in theory, all tests are important. ;)